src/pages/Register.js

```diff
@@ -105,11 +159,77 @@ const Register = () => {
                 id,
                 value: email,
                 onChange: (v) => setEmail(v),
-                style: { width: '50ex' },
+                style: { width: '66ex' },
+              }),
+            ]),
+          ]),
+      ]),
+      h(IdContainer, [
+        (id) =>
+          div({ style: { lineHeight: '170%' } }, [
+            label({ htmlFor: id, style: { display: 'block', marginTop: '2rem' } }, ['Organization *']),
```

Comment:
Note that there is also a component called `FormLabel` that handles displaying the asterisk for required fields. It would make it pretty easy for you to _not_ display the asterisk when the fields aren't actually required. But then you should change all the required labels in this view so that they render consistently.

```
export const FormLabel = ({ style = {}, required = false, children, ...props }) => {
  return label({ ...props, style: { ...styles.formLabel, ...style } }, [children, required && ' *']);
};

Reply:
Converted all the inputs to use `FormLabel`!